lightly/active_learning/utils/keypoint_detection_output.py

```diff
@@ -0,0 +1,60 @@
+""" Keypoint """
+from typing import Union, Tuple, List
+
+import numpy as np
+
+class KeypointDetection:
+    """Class which represents all keypoints of one object.
+
+    Attributes:
+        keypoints:
+            Predicted keypoints in the [x0, y0, c0, ... xk, yk, ck] format.
+            Thus the shape of the array for k keypoints must be (k*3, ).
+        object_id:
+            The id of the predicted object, e.g. "3" denoting a person.
+            This is not used at the moment.
+
+    Examples:
+        >>> # Create the representation of wo keypoints with confidences
+        >>> # of 0.8 and 0.1 respectively.
+        >>> keypoints = [334, 534, 0.8, 456, 432, 0.1]
+        >>> keypoints = np.array(keypoints)
+        >>> keypoint_detections = KeypointDetection(keypoints, 3)
+    """
+
+    def __init__(self, keypoints: np.ndarray, object_id: int = -1):
```

Comment:
is it required that the keypoints are an `np.ndarray`? They could also just be a list which is the more flexible typehint, no?

Reply:
I changed it to a list.